scripts/README_INSIDER_BUILD.md

```diff
@@ -23,7 +23,8 @@ Read more about [how to contribute to the Prisma VSCode extension](./packages/vs
 
 ## Telemetry
 
-This extension collects telemetry data to help us better the usage of the extension. You can read more about that [here](https://www.prisma.io/docs/reference/more/telemetry).
+This extension collects telemetry data to help us better the usage of the extension. You can read more about that [here](https://www.prisma.io/docs/reference/more/telemetry).\
+The extension respects the `telemetry.enableTelemetry` setting in VSCode.
```

Comment:
Probably some instructions on how to set it/possible values (is it boolean or does it respect some sort of "truth-y" values)

Reply:
Yes ,good point, I've added a comment.